test/tests/test_environments/test_java_xml_env.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
You have to add this test to https://github.com/fission/fission/blob/5c09099084ba0983d62ea3bb71209c40bdb9c71c/test/test_utils.sh#L516-L522 so that the CI will run this test.